hack/ginkgo-e2e.sh

```diff
@@ -102,6 +102,7 @@ export PATH=$(dirname "${e2e_test}"):"${PATH}"
   --repo-root="${KUBE_ROOT}" \
   --node-instance-group="${NODE_INSTANCE_GROUP:-}" \
   --prefix="${KUBE_GCE_INSTANCE_PREFIX:-e2e}" \
+  --os-distro="${OS_DISTRIBUTION:-debian}" \
```

Comment:
> does jenkins running GKE tests use the env variable OS_DISTRIBUTION by default?

`OS_DISTRIBUTION` has a default value of `debian`. Only Trusty e2e jobs reset it AFAIK, through `KUBE_OS_DISTRIBUTION`.

https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/config-test.sh#L38